cmd/start.go

```diff
@@ -161,32 +133,24 @@ func Start(c *cli.Context) error {
 	}
 	persistence := persistence.NewEncryptedPersistence(
 		handle,
-		config.Ethereum.Account.KeyFilePassword,
+		extractKeyFilePassword(config),
 	)
 
-	sanctionedApplications, err := config.SanctionedApplications.Addresses()
-	if err != nil {
-		return fmt.Errorf("failed to get sanctioned applications addresses: [%v]", err)
-	}
-
 	clientHandle := client.Initialize(
 		ctx,
-		operatorPublicKey,
-		ethereumChain,
+		operatorKeys.public,
+		chainHandle,
 		networkProvider,
 		persistence,
-		sanctionedApplications,
```

Comment:
`sanctionApplications` is the main source of conflicts in this file. I saw that master doesn't use it anymore, so I removed it here, and in `client.go`